package.json

```diff
@@ -87,7 +87,8 @@
     "json-schema": "^0.4.0",
     "lodash.memoize": "^4.1.2",
     "lodash.merge": "4.6.2",
-    "lodash.mergewith": "4.6.2"
+    "lodash.mergewith": "4.6.2",
+    "perron": "^0.11.4"
```

Comment:
need to be changed to dev dependency in case someone would want to launch my script locally